gulpfile.js

```diff
@@ -170,6 +170,7 @@ gulp.task('pot', function () {
     return gulp.src([
             'openslides/core/static/templates/*.html',
             'openslides/*/static/templates/*/*.html',
+            'openslides/*/static/templates/*/*/*.html',
```

Comment:
We should try `'openslides/*/static/templates/**/*.html'`. This is possbily more generic.